cs-dropbox/src/main/java/io/cloudslang/content/dropbox/entities/dropbox/SuffixUri.java

```diff
@@ -16,7 +16,10 @@
  * 5/30/2017.
  */
 public enum SuffixUri {
-    CREATE_FOLDER("CreateFolder", "/create_folder");
+    CREATE_FOLDER("CreateFolder", "/create_folder"),
+    CREATE_FOLDER_V2("CreateFolder2", "/create_folder_v2"),
+    DELETE_FILE_OR_FOLDER("DeleteFileOrFolder", "/delete"),
+    DELETE_FILE_OR_FOLDER_V2("DeleteFileOrFolder2", "/delete_v2");
```

Comment:
yes... is official and it goes like this: deprecated api will consume "/create_folder" uri and the actual api will consume "/create_folder_v2". See here: https://www.dropbox.com/developers/documentation/http/documentation#files-create_folder